price_service/sdk/js/.eslintrc.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
   rules: {
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }]
```

Comment:
Please remove this config, we can add eslint-ignore lines if there are any intentionally unused vars.